apps/indexer/lib/indexer/block/catchup/massive_blocks_fetcher.ex

```diff
@@ -56,13 +56,19 @@ defmodule Indexer.Block.Catchup.MassiveBlocksFetcher do
   end
 
   defp process_block(block_fetcher, number) do
-    case Fetcher.fetch_and_import_range(block_fetcher, number..number, %{timeout: :infinity}) do
-      {:ok, _result} ->
-        Logger.info("MassiveBlockFetcher successfully proceed block #{inspect(number)}")
-        MassiveBlock.delete_block_number(number)
-        []
-
-      {:error, error} ->
+    try do
+      case Fetcher.fetch_and_import_range(block_fetcher, number..number, %{timeout: :infinity}) do
+        {:ok, _result} ->
+          Logger.info("MassiveBlockFetcher successfully proceed block #{inspect(number)}")
```

Comment:
I guess there is a typo in log message

```suggestion
          Logger.info("MassiveBlockFetcher successfully processed block #{inspect(number)}")
```